### PR TITLE
add iwara

### LINKED ISSUE #7107: 肯请大佬帮忙制作RSS https://ecchi.iwara.tv/
### 网站地址
https://ecchi.iwara.tv/
### 网站描述
MMD视频网站
### 需要生成什么内容？
指定用户发布的视频及下载地址

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #7107

## 完整路由地址 / Example for the proposed route(s)

```routes
/iwara/users/kelpie/video
/iwara/users/kelpie/image
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [X] New Route
- [X] Documentation
  - [X] CN
  - [X] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — CHANGES_REQUESTED** [review]
~~Missing en docs~~ 
EDIT: Fixed

File: lib/v2/iwara/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+};
+
+const selectorMap = {
+    video: 'div.node.node-video.node-teaser.node-teaser.clearfix',
+    image: 'div.node.node-image.node-teaser.node-teaser.clearfix',
+};
+
+module.exports = async (ctx) => {
+    const username = ctx.params.username;
+    const type = ctx.params.type || 'video';
```
Comment (TonyRL) [inline lib/v2/iwara/index.js:18]:
```suggestion
    const type = ctx.params.type ?? 'video';
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/anime.md [hunk 2]
```diff
@@ -253,6 +253,18 @@ pageClass: routes
 
 </Route>
 
+## iwara
+
+### 用户
+
+&lt;Route author="Fatpandac" example="/iwara/users/kelpie/video" path="/iwara/users/:username/:type?" :paramsDesc="['', 'type 默认为 video'] radar="1" rssbud="1">
```
Comment (TonyRL) [inline docs/anime.md:260]:
```suggestion
<Route author="Fatpandac" example="/iwara/users/kelpie/video" path="/iwara/users/:username/:type?" :paramsDesc="['', 'type 默认为 video'] radar="1" rssbud="1">
```

File: lib/v2/iwara/index.js (hunk 1 above)
Comment (TonyRL) [inline lib/v2/iwara/index.js:18]:
```suggestion
    const type = ctx.params.type ?? 'video';
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/anime.md (hunk 2 above)
Comment (TonyRL) [reply docs/anime.md:260]: Please provide description to `username`

File: lib/v2/iwara/index.js (hunk 1 above)
Comment (Fatpandac) [reply lib/v2/iwara/index.js:18]: Thanks

File: docs/anime.md (hunk 2 above)
Comment (Fatpandac) [reply docs/anime.md:260]: Thx. I'll add a description for `username` late.

File: docs/en/anime.md [hunk 3]
```diff
@@ -59,3 +59,15 @@ For example: https://www.webtoons.com/zh-hant/drama/gongzhuweimian/list?title_no
...
+
+### User
+
+<Route author="Fatpandac" example="/iwara/users/kelpie/video" path="/iwara/users/:username/:type?" :paramsDesc="['username, can find in userpage', 'video by default']" radar="1" rssbud="1">
+
+| type | video | image |
+| :--: | :---: | :---: |
+|      | video | image |
+
+</Route>
```
Comment (TonyRL) [inline docs/en/anime.md:73]:
```suggestion
<RouteEn author="Fatpandac" example="/iwara/users/kelpie/video" path="/iwara/users/:username/:type?" :paramsDesc="['username, can find in userpage', 'video by default']" radar="1" rssbud="1">

| type | video | image |
| :--: | :---: | :---: |
|      | video | image |

</RouteEn>
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/en/anime.md (hunk 3 above)
Comment (Fatpandac) [reply docs/en/anime.md:73]: thx.

**Pull request merged**